E
======================================================================
ERROR: test_untitled (__main__.Untitled)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:/BARANCEV/Seleniun_php4dvd_lesson2_Rovinsky/php4dvd/php4dvd_positive.py", line 36, in test_untitled
    driver.find_element_by_id("submit").click()
  File "C:\BARANCEV\Seleniun_php4dvd_lesson2_Rovinsky\ENV\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 234, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File "C:\BARANCEV\Seleniun_php4dvd_lesson2_Rovinsky\ENV\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 712, in find_element
    {'using': by, 'value': value})['value']
  File "C:\BARANCEV\Seleniun_php4dvd_lesson2_Rovinsky\ENV\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 201, in execute
    self.error_handler.check_response(response)
  File "C:\BARANCEV\Seleniun_php4dvd_lesson2_Rovinsky\ENV\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 188, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"id","selector":"submit"}


----------------------------------------------------------------------
Ran 1 test in 41.091s

FAILED (errors=1)

Process finished with exit code 1
